include/ttmlir/Dialect/TTIR/Analysis/TTIRAnalysis.h

```diff
@@ -10,19 +13,20 @@ namespace mlir::tt::ttir {
 template <class I, class R> class TTIRAnalysis {
 protected:
   Operation *op;
-  bool is_valid;
+  bool is_valid = false;
   R analysis_result;
   I analysis_input;
 
-  TTIRAnalysis(Operation *op) : op(op), is_valid(false) {}
```

Comment:
I thought I have run all the pre-commit checks, which checks were failing because of my change?

Reply:
Yeah I see they are reenabled now after open sourcing.